subsys/logging/log_core.c

```diff
@@ -478,6 +492,69 @@ u32_t log_filter_get(struct log_backend const *const backend,
 	}
 }
 
+static void *log_malloc(u32_t len)
```

Comment:
Wasn't the plan to make this public as well? For the `bt_addr_le_to_str()` case we wouldn't then need to use an intermediate string buffer to call `log_strdup()` on, rather the string could be directly encoded into a memory region provided by log_malloc().

Reply:
See updated PR message, i've explained there why i've removed that from API

Reply:
Ok, so we'll need a stack buffer and log_strdup() then. Thanks for the explanation.